src/general/Species.cs

```diff
@@ -119,7 +119,9 @@ public Color GUIColour
     ///   with bolding.
     /// </summary>
     [JsonIgnore]
-    public string FormattedNameBbCode => PlayerSpecies ? $"[b][i]{FormattedName}[/i][/b]" : $"[i]{FormattedName}[/i]";
+    public string FormattedNameBbCode => PlayerSpecies ?
+        $"[url=species:{GUICommon.ConvertToSafeMetaTag(StringCode)}][b][i]{FormattedName}[/i][/b][/url]" :
```

Comment:
Storing the entire serialised class in the meta tag feels really wrong but I don't think there's a much simpler way to do it, since (like the Thriveopedia links) the CustomRichTextLabel could be used anywhere, even outside of games in progress where species are accessible. Please let me know if there's a better way.

Reply:
This can be a ton of data that the bbcode parser needs to get through each time it is updated. I think a better solution would be to just have the species internal id here and maybe have the thriveopedia mediate the request for the species data? Each species should have a Thriveopedia page about it in the future so that kind of design would make sense to make it a requirement for the thriveopedia to have info on all species data.